.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```

Comment:
What tool consumes these files?

Reply:
the setup-ruby would consume this (i had some issue putting it into review-cask-pr folder, which is kind of a bug in setup-ruby action)

Reply:
gonna merge for now, we can discuss/revisit if needed